services/kaze/controllers/helper.go

```diff
@@ -47,6 +49,29 @@ func fetchInstances(c *gin.Context, instance string) {
 	})
 }
 
+// FetchAllInstancesByUser returns all instances owned by a user
+func FetchAllInstancesByUser(c *gin.Context) {
+	filter := utils.QueryToFilter(c.Request.URL.Query())
+	claims := middlewares.ExtractClaims(c)
+	if claims == nil {
+		utils.SendServerErrorResponse(c, errors.New("Failed to extract JWT claims"))
+		return
+	}
+	filter[mongo.OwnerKey] = claims.GetEmail()
+
+	projection := bson.D{
+		{"name", 1},
+		{"language", 1},
+		{"instance_type", 1},
+	}
```

Comment:
Replace this with
```
projection :=  types.M{
		{mongo.NameKey, 1},
		{mongo.LanguageKey, 1},
		{mongo.InstanceTypeKey, 1},
	}
```

This will also resolve the errors faced with travis-ci